src/atoms/feedCardItem/types.ts

```diff
@@ -0,0 +1,61 @@
+export interface TransactionData {
+  hash: string;
+  timestamp: number;
+  block: number;
+  wallet: string;
+  from: string;
+  to: string;
+  tx_hash: string;
+  tx_type: string;
+  dex: '0x'
+  | '1inchV3'
+  | '1inchV4'
+  | 'BalancerV2'
+  | 'Bancor'
+  | 'CurveV2'
+  | 'MetaMask'
+  | 'ParaSwapP4'
+  | 'ParaSwapP5'
+  | 'SushiSwap'
+  | 'UniswapV2'
+  | 'UniswapV3'
+  ;
+  token0_address: string;
+  token0_name: string;
+  token0_symbol: string;
+  token0_amount: number,
+  token0_amount_usd: number;
+  token0_price_usd: number;
+  token1_address: string;
+  token1_name: string;
+  token1_symbol: string;
+  token1_amount: number;
+  token1_amount_usd: number;
+  token1_price_usd: number;
+  index: number;
+  chain:
```

Comment:
Ah rather than typing all the keys out you can just do `keyof typeof <obj>` and it will be the exact same type as you have here right now